misc/package/Data Files/MWSE/core/mcm/components/settings/Slider.lua

```diff
@@ -101,6 +147,7 @@ function Slider:enable()
 
 	-- But we want the label to update in real time so you can see where it's going to end up
 	self.elements.slider:register(tes3.uiEvent.partScrollBarChanged, function(e)
+		self:updateVariableValue()
```

Comment:
There is `mwseMCMConfigVariable` which writes to the mod's config json file on every variable change, so writing to variable value could be laggy. Is this line necessary for it to work properly?

Completely unrelated to this, we should probably deprecate `mwseMCMConfigVariable` because of the issue. I searched the Lua mods dump and found no usage of `createConfigVariable`.

Reply:
oh i forgot about `mwseMCMConfigVariable`s. the call to `updateVariableValue` isn't really necessary, it would just mean that the `updateValueLabel` method would have to do its computation after calling `convertToVariableValue(self.elements.slider.widget.current)`. i added it because i thought it made more sense to update the variable value since the `convertToVariableValue` function had to be called anyway, and it some sense it felt "more natural" to update the label based on the current variable value, but the  `mwseMCMConfigVariable` thing complicates this a bit. i guess also from an end-user perspective it seems a bit "unintuitive" that moving a slider wouldn't update the variable value, but it probably doesn't come up very often. and having a config variable write to file several times a second would not be ideal.

 i can change it if you'd like or keep it as it is. i dont have a strong preference either way

Reply:
It makes sense to update the variable's value here. We can remove this if it proves to cause issues.